http/netty/components/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/sender/TargetHandler.java

```diff
@@ -123,7 +132,7 @@ protected void readTimedOut(ChannelHandlerContext ctx) {
         ctx.channel().close();
 
         if (targetChannel.isRequestWritten()) {
-            String payload = "<errorMessage>" + "ReadTimeoutException occurred for endpoint" + targetChannel.
+            String payload = "<errorMessage>" + "ReadTimeoutException occurred for endpoint " + targetChannel.
```

Comment:
Isn't it wrong to return an XML here? The incoming request could have been JSON, or the Accept header could be application/json. In such a case, it is fundamentally wrong to return XML.